addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -185,7 +185,11 @@ function makeActionManager(env) {
      * with a unique jsId.
      */
     function _preprocessAction(action, context = {}) {
-        action._originalAction = JSON.stringify(action);
+        try {
+            action._originalAction = JSON.stringify(action);
+        } catch (_e) {
+            // do nothing, the action might simply not be stringified (e.g. if it contains cycles)
```

Comment:
`the action might simply not be stringified` => `the action might simply not be serializable`